src/languageservice/services/yamlCompletion.ts

```diff
@@ -71,6 +71,7 @@ export class YamlCompletion {
   private configuredIndentation: string | undefined;
   private yamlVersion: YamlVersion;
   private indentation: string;
+  private prefixIndentation = '';
```

Comment:
Should this get a better name to indicate that this prefix is used for arrays only.

Reply:
I thought that this property can be used for very similar functionality in the future:
colon without space

```yaml
simpleProp:#invoke here without space
```

but I can change the name to `arrayPrefixIndentantion` and when this functionality is implemented in the future we can change it back to something more generic.

What do you think?

Reply:
Unless you are planning to provide a PR for it soon. I would prefer descriptive names.

Reply:
Not sure when, so I renamed it.